Makefile

```diff
@@ -155,6 +155,17 @@ else
 JERRYLIB=outdir/$(BOARD)/libjerry-core-parser.a
 endif
 
+.PHONY: flash
+flash:  analyze generate $(JERRYLIB) $(ARC)
+	@make -f Makefile.zephyr flash	BOARD=$(BOARD) \
+									VARIANT=$(VARIANT) \
```

Comment:
For the *standard* tab size of 8 this looks hugely right-indented (easily visible on github).